tests/test_visitors/test_ast/test_annotations/test_annotation_metadata/conftest.py

```diff
@@ -7,6 +7,22 @@ def function(arg: {0}):
     ...
 """
 
+function_arg_alias_template = """
+Type_Alias = {0}
+
+def function(arg: Type_Alias):
+    ...
+"""
+
+function_arg_multiple_alias_template = """
+a, Another_Alias = {0}, str
+b = c = {0}
```

Comment:
Sorry, this does not look correct to me. 

We need to make three separate new templates for this case:
1. `a = {0}`
2. `b = c = {0}`
3. `d, e = {0}, {0}`

And nothing else, function is redundant here. 

I am also not sure about how `d, e = {0}, {0}` should behave. I guess, that it should raise two violations. However, it is not critical and one is good enough.
See which implementation is easier.

Thanks a lot for your time and effort! 👍

Reply:
Will do.

But with `d, e = Literal['a'], Literal['b']` I have a false positive `NestedAnnotationsViolation`. Not sure how to defeat it.

Reply:
Well, regex will always help 😁
Fixed.